src/app.js

```diff
@@ -155,9 +155,10 @@ class Messenger extends EventEmitter {
 
   start() {
     const port = config.get('port');
+    this.emit('app.starting', {port});
     this.app.listen(port, (err) => {
       if (err) throw err;
-      debug('Server running on port %s', port);
```

Comment:
I find this log statement useful, and I don't want to add it back on all the sdk consumers. It's opt in anyways.